src/btreemap/proptests.rs

```diff
@@ -231,5 +235,27 @@ fn execute_operation<M: Memory>(
                 assert_eq!(v1, &v2);
             }
         }
+        Operation::PopLast => {
+            assert_eq!(std_btree.len(), btree.len() as usize);
+
+            let idx = std_btree.len();
+
+            if let Some(entry) = btree.iter().skip(idx).take(1).next() {
+                eprintln!("PopLast");
+                assert_eq!(std_btree.pop_last(), Some(entry.clone()));
+                assert_eq!(btree.pop_last(), Some(entry));
+            }
```

Comment:
I still don't understand why this isn't just:

```suggestion
            eprintln!("PopLast");
            assert_eq!(std_btree.pop_last(), btree.pop_last());
```